email-notification-service/src/db.ts

```diff
@@ -126,10 +126,10 @@ class DbConnector {
     return "";
   };
 
-  private initializeConnection = (): knex => {
+  private initializeConnection = (): Knex => {
     logger.info("Initialize database connection");
     logger.info(config);
-    const knexConfig: knex.Config = {
+    const knexConfig = {
```

Comment:
is there no config type anymore?

Reply:
Yes there is.

I had to experiment with the types a bit, this one got lost in the process. I pushed again with Knex.Config Type